### PR TITLE
add specific info on how to access the dashboards

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary

Fixes: #15924

Added information on how to access the dashboards based on some feedback we got [here](https://github.com/netdata/netdata-cloud/issues/912)

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,10 @@ It scales nicely from just a single server to thousands of servers, even in comp
 
   Check also the [Netdata Deployment Strategies](https://learn.netdata.cloud/docs/architecture/deployment-strategies) to decide how to deploy it in your infrastructure.
 
+  By default, you will have immediately available a local dashboard. Netdata starts a web server for its dashboard at port `19999`. Open up your web browser of choice and
+navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If you're unsure, try
```
Comment (ilyam8) [inline README.md:169]: What does it mean "If you're unsure"? I think we can safely assume that the person who can install Netdata knows the host's IP address (or hostname).

Comment (hugovalente-pm) [reply README.md:169]: @ilyam8 I copied it from [here](https://github.com/netdata/netdata/blob/31924c0eb2fb245bdf047a5f4fe9f5ef10baf24e/web/gui/README.md?plain=1#L16), from what it is to work just a fallback suggestion don't actually see a harm keeping it here

Comment (ilyam8) [reply README.md:169]: I see the harm - the user can open different Netdata (local) after installation on the remote host.

Comment (ilyam8) [reply README.md:169]: My suggestion is to remove the sentence or change the wording - not "if unsure" but if installed on Localhost.

Comment (hugovalente-pm) [reply README.md:169]: got it, changed it to
> If installed on localhost, you can access it through `http://localhost:19999`.

File: docs/category-overview-pages/accessing-netdata-dashboards.md [hunk 2]
```diff
@@ -23,5 +23,17 @@ The Netdata dashboard consists of the following main sections:
...
+
+### Netdata Cloud
+
+You can access the dashboard at https://app.netdata.cloud/ and [sign-in](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/sign-in.md) with an account or [sign-up](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/sign-in.md#dont-have-a-netdata-cloud-account-yet) if you don't have an account yet.
+
+### Netdata Agent
+
+Netdata starts a web server for its dashboard at port `19999`. Open up your web browser of choice and
+navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If you're unsure, try
+`http://localhost:19999` first.
```
Comment (ilyam8) [inline docs/category-overview-pages/accessing-netdata-dashboards.md:36]:
```suggestion
navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If installed on localhost, you can access it through `http://localhost:19999`.
```

Comment (hugovalente-pm) [reply docs/category-overview-pages/accessing-netdata-dashboards.md:36]: this is already on this commit. right?

https://github.com/netdata/netdata/pull/15925/commits/4097f8685482198b319d104e3afe3c3440940157#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R169

if you are ok lets resolve the conversation

Comment (ilyam8) [reply docs/category-overview-pages/accessing-netdata-dashboards.md:36]: It is not, you haven't changed `docs/category-overview-pages/accessing-netdata-dashboards.md`.

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**